### DISCUSSION TITLE
Singlephase/Split-Phase Data Per Line

### DISCUSSION THREAD

**Discussion by gutowscr471** [jertel/vuegraf#123] [General]
Is there anyway to see the data on ports A and B via VueGraf.  I have split phase electric in the U.S and would like to see if my Port A and Port B readings are near balanced for my loads.  The only way I can see this now is data download from Emporia (sample below).

<img width="405" alt="image" src="https://github.com/jertel/vuegraf/assets/121516947/c119eedc-e860-4ecb-b678-8048f13c898c">

**Comment by jertel** [comment]
Does your reference to "ports A and B" represent the two panel main clamps? If so Vuegraf shows the combined usage and does not have the individual metrics for each clamp.

**Reply by gutowscr471** [reply]
Correct, A and B are my two main panel clamps.  Was hoping to see what each clamp is reading over time (gives a sense of how unbalanced each leg is of your panel for off-grid solar sizing).  For example, a 120v microwave and coffee maker on the same leg could be a 30amp (3600w) imbalance will would cause a fault in an off-grid solar inverter.

**Reply by jertel** [reply]
Vuegraf pulls usage data but this thread on the underlying API lib suggest it might be possible, if you wanted to write your own Python app: https://github.com/magico13/PyEmVue/issues/22.

A more recent thread with your same question was left unanswered, back in December: https://github.com/magico13/PyEmVue/issues/44
